3party/osrm/osrm-backend/plugins/MapsMePlugin.hpp

```diff
@@ -29,6 +29,64 @@
 #include <string>
 #include <vector>
 
+double constexpr kMaxDistanceToFindMeters = 1000.0;
+
+using TMapRepr = pair<size_t, m2::PointD>;
+
+class UsedMwmChecker
```

Comment:
I'd like leave this class here. I want to minimize links to omim based libraries.